drivers/stepper/adi_tmc/adi_tmc_spi.c

```diff
@@ -76,7 +60,7 @@ int tmc_spi_read_register(const struct spi_dt_spec *bus, const uint8_t read_addr
 		((uint32_t)rx_buffer[3] << 8) + (uint32_t)rx_buffer[4];
 
 	print_tx_rx_buffer(tx_buffer, rx_buffer);
```

Comment:
Does it make sense to even drop this? I assume, on the spi layer this would be logged?

Reply:
I guess drop it, but i remember it correctly, the SPI logs would only show the data buffer address (not sure).
But I would say, we can drop this.